shacl-compact-syntax/scope.md

```diff
@@ -18,34 +18,67 @@ Note that this should be done *after* these features have been supported by the
 
 ## New Features
 
-### Support for additional SHACL-C shorthands
- - Support Rules
- - Support (SPARQL) Functions
- - Support shorthand (`<`, `=`, `<=`, etc.) for property pair constraints
- - [Support `xone` and more `or` cases](https://github.com/w3c/shacl/issues/12)
- - Opting out of the production of the triple `?baseUri rdf:type owl:Ontology`
-  - Support for non-validating characteristics (e.g., generating `sh:order` triples by declaring `@order` at the top of a file/shape, allow grouping by `@group` at the top of a set of properties, and allowing a `@description` above properties).
+### Support for Additional SHACL-C Constructs
+- shorthand (`<`, `=`, `<=`, etc.) for property pair constraints
+- [Support `xone` and more `or` cases](https://github.com/w3c/shacl/issues/12)
+- Opt out of the production of the triple `?baseUri rdf:type owl:Ontology`
+- Non-validating characteristics (e.g., generating `sh:order` triples by declaring `@order` at the top of a file/shape, allow grouping by `@group` at the top of a set of properties, and allowing a `@description` above properties).
+- Target shapes, see [rsx:targetShape](https://rdf4j.org/shacl/extensions.html)
 
-The first 2 of these have been supported in [this xtext file](https://gitlab.com/allotrope-open-source/shape-editor/-/blob/master/src/com.osthus.shapes.shaclc.parent/com.osthus.shapes.shaclc/src/com/osthus/shapes/shaclc/SHACLC.xtext) by [allotrope](https://www.allotrope.org/). Some or all of these additions may be ignored in favour of keeping SHACL-C a simple syntax to implement and use.
+A number of features from [SHACLC.xtext](https://gitlab.com/allotrope-open-source/shape-editor/-/blob/master/src/com.osthus.shapes.shaclc.parent/com.osthus.shapes.shaclc/src/com/osthus/shapes/shaclc/SHACLC.xtext) by [Allotrope](https://www.allotrope.org/):
+- [Shape Libraries](https://rawgit2.com/VladimirAlexiev/shacl/shaclc-grammars/shacl-compact-syntax/grammar/shaclc-XText.html#ShaclDoc) (versioned). Packaging shapes into libraries enables modularized development of shapes. 
```

Comment:
Could you please provide additional information or a pointer to describe what triples are produced when parsing this part of the grammar to make it clearer how the shape library works.

Reply:
I have no idea whether these are implemented. They are good material for extending the scope of SHACL 1.2. I'll make a separate issue. 
Guess we should add a note which are part of SHACL, and which are speculative?

Reply:
>  Guess we should add a note which are part of SHACL, and which are speculative?


That would be great :)

Reply:
ok: [move speculative features into subsection ### Extra SHACL Features](https://github.com/w3c/shacl/pull/70/commits/ec49e88b7da503258c41a0e1cbbd5229760068f3)